scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -285,8 +285,15 @@ object Term {
     def value = ""
     checkParent(ParentChecks.AnonymousImport)
   }
+  @branch
+  trait SelectLike extends Term.Ref with Pat {
+    def qual: Term
+    def name: Term.Name
+  }
+  @ast
+  class Select(qual: Term, name: Term.Name) extends SelectLike
   @ast
-  class Select(qual: Term, name: Term.Name) extends Term.Ref with Pat
+  class SelectPostfix(qual: Term, name: Term.Name) extends SelectLike
```

Comment:
We should put that into breaking section when releasing as people should update from just Select to SelectLike

Reply:
can you explain what you mean here? are you talking about release notes, or about the code itself?

Reply:
I just mean we should have a special section in the release notes about potential breaking changes people could encounter since their code will now parse differently